tests/training/test_online_strategies.py

```diff
@@ -0,0 +1,105 @@
+import os
+import sys
+import unittest
+
+from torch.nn import CrossEntropyLoss
+from torch.optim import SGD
+
+from avalanche.logging import TextLogger
+from avalanche.models import SimpleMLP
+from avalanche.benchmarks.scenarios.online_scenario import OnlineCLScenario
+from avalanche.training import OnlineNaive
+from tests.unit_tests_utils import get_fast_benchmark
+
+
+class StrategyTest(unittest.TestCase):
+    if "FAST_TEST" in os.environ:
+        fast_test = os.environ["FAST_TEST"].lower() in ["true"]
+    else:
+        fast_test = False
+    if "USE_GPU" in os.environ:
+        use_gpu = os.environ["USE_GPU"].lower() in ["true"]
+    else:
+        use_gpu = False
+
+    print("Fast Test:", fast_test)
+    print("Test on GPU:", use_gpu)
+
+    if use_gpu:
+        device = "cuda"
+    else:
+        device = "cpu"
+
+    def init_sit(self):
+        model = self.get_model(fast_test=True)
+        optimizer = SGD(model.parameters(), lr=1e-3)
+        criterion = CrossEntropyLoss()
+        benchmark = self.load_benchmark(use_task_labels=False)
+        benchmark_streams = benchmark.streams.values()
+        ocl_benchmark = OnlineCLScenario(benchmark_streams)
+        return model, optimizer, criterion, ocl_benchmark
+
+    def test_naive(self):
+        # SIT scenario
+        model, optimizer, criterion, my_nc_benchmark = self.init_sit()
+        strategy = OnlineNaive(
```

Comment:
can you test with and without boundaries? we need both.

Instead, I don't think testing multi-task is useful here. You can remove it if you want

Reply:
Ok, will add w/o task boundaries as well.

What do you mean by multi-task not being useful here?

Reply:
> Instead, I don't think testing multi-task is useful here. You can remove it if you want

I see what you mean now, that's a mistake :D. Will remove it!